### DISCUSSION TITLE
erro no github desktop

### DISCUSSION THREAD

**Discussion by AmericoLanga** [community/community#48683] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

Ajuda

Fatal:   bad config line 1 in file C:/users/Americo-langa/. gitconfig

**Comment by SilasRodrigues19** [comment]
E ai @AmericoLanga 

Você editou esse arquivo por engano? Esse arquivo é o arquivo padrão de configuração quando se instala o Git na máquina.

Alguma coisa desconfigurou e ele está dando erro, geralmente são as configurações de usuário, você pode tentar corrigir assim:

```bash
git config --global user.name "SeuUsuario"
git config --global user.email seuemail@domain.com
```

Só lembre-se de substituir pelo seu usuário e seu e-mail nos comandos acima, caso isso não resolva ao invés de você ficar tentando achar o problema você pode reinstalar o Git e configurar o usuário novamente.

**Reply by AmericoLanga** [reply]
valeu mano, vou tentar Agora!

**Comment by matheusbastosnhoncanse** [comment]
Pra mim não consegui, mesmo com as duas tecnicas
